Src/Sound/SoundResources.cpp

```diff
@@ -0,0 +1,118 @@
+// the Sound Resources object is similar to GraphicsResources and handles sound layers.
+
+#include "SoundResources.h"
+
+
+class SoundStream;
+
+SoundStream* x;
```

Comment:
Global variables in C++ are disallowed. Our exceptions to this rule are all caused by restrictions in some of our libraries, and we try to implement them in such a way that there is still as little global state as possible.

Reply:
Testing purposes.
These files will likely be re-written almost entirely.